src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/AddSABONumberModal.tsx

```diff
@@ -58,12 +58,12 @@ const AddSABONumberModal = ({ modalShow, onHide, reimbursementRequestId }: AddSA
       handleUseFormSubmit={handleSubmit}
       onFormSubmit={onSubmit}
       disabled={!isValid}
-      formId="add-sabo-number"
+      formId="add-SABO-number"
     >
       <FormControl>
         <ReactHookTextField
           control={control}
-          name="saboNumber"
+          name="SABONumber"
```

Comment:
We don't have to capitalize the sabo text here since `formId `and `name `fields represent the component id and name. We only want to capitalize the sabo text that gets displayed on the webpage, so we can undo the changes in this file.